src/libcore/sync/atomic.rs

```diff
@@ -149,21 +159,26 @@ unsafe impl<T> Sync for AtomicPtr<T> {}
 #[derive(Copy, Clone, Debug)]
 pub enum Ordering {
     /// No ordering constraints, only atomic operations. Corresponds to LLVM's
-    /// `Monotonic` ordering.
+    /// [`Monotonic`][1] ordering.
+    /// [1]: http://llvm.org/docs/Atomics.html#monotonic
```

Comment:
While you're here, would you mind helping with a minor formatting issue as well? This should look like this:

```rust
     /// No ordering constraints, only atomic operations.
     ///
     /// Corresponds to LLVM's [`Monotonic`] ordering.
     ///
     /// [`Monotonic`]: http://llvm.org/docs/Atomics.html#monotonic
```

Thanks 😄